addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/AddNodeMessageClass.java

```diff
@@ -39,8 +39,9 @@
     private final int ADD_NODE_STATUS_FAILED = 0x07;
 
     private final int OPTION_HIGH_POWER = 0x80;
+    private final int OPTION_NETWORK_WIDE = 0x40;
 
-    public SerialMessage doRequestStart(boolean highPower) {
+    public SerialMessage doRequestStart(boolean highPower, boolean networkWide) {
         logger.debug("Setting controller into INCLUSION mode.");
```

Comment:
Since we now have more options, can you enhance the logging here to print the two options.